docs/index_es.html

```diff
@@ -0,0 +1,118 @@
+          <li>Copia/pega fragmentos de código de Stack Overflow, Glitch, Codepen o donde sea que encuentren respuestas.</li>
+          <li>Da crédito donde se debe crédito.</li>
+          <li>Creates community and shares knowledge.</li>
+          <li>Dedica tiempo a cosas fuera de la ingeniería, como pasatiempos, amigos y familia.</li>
+          <li>Tiene un horario que les permite mantener un equilibrio saludable entre el trabajo y la vida y también respeta los límites de tiempo de los demás.</li>
+          <li>No se mide por puntajes de contribución arbitrarios en ningún sitio web, y tampoco juzga a otras personas por los suyos.</li>
+          <li>Escribe código que <span title="No se preocupe por enviar un PR para esto, es un error typo intencional, ya sabe, ¡porque creamos errores!">&emdash;</span> <em>gasp</em> &emdash; tiene bugs.</li>
+          <li>Escribe código que otro/as pueden leer.</li>
+          <li>Lee los documentos.</li>
+          <li>Actualiza los documentos.</li>
+          <li>No es necesario que le apasione el código que escribe o los problemas que resuelve, pero puede ser que si.</li>
+          <li>No se sorprende cuando alguien no sabe algo.</li>
+          <li>Está dispuesto/a y es capaz de colaborar con otros.</li>
+          <li>Celebra públicamente a otro/as por sus victorias.</li>
+          <li>Hace preguntas antes de proporcionar comentarios críticos.</li>
+          <li>Da retroalimentación dura en privado.</li>
+          <li>Trata a otras personas como a <em>el/ella</em> le gustaría ser tratado/a.</li>
+          <li>Proporciona revisiones de código y retroalimentación a sus compañero/as que son constructivos, útiles y presentados con tacto, ayudando a sus compañero/as a crecer personal y profesionalmente.</li>
+          <li>Expresa su aprecio por las revisiones de código y las retroalimentaciones de sus compañero/as que son constructivos y útiles.</li>
+          <li>A veces se siente herido/a por la retroalimentación crítica, pero no reacciona destructivamente.</li>
+          <li>A veces pasa horas depurando un problema menor y lo soluciona en unos minutos.</li>
+          <li>A veces toma breves descansos para despejarse la cabeza.</li>
+          <li>A veces se siente frustrado/a por su entorno y necesita desahogarse.</li>
+          <li>Comete errores de vez en cuando y encuentra crecimiento en esos errores.</li>
+          <li>Están dispuesto/as a admitir cuando están equivocado/as y no tienen miedo de decir "No lo sé".</li>
+          <li>Puede que le guste o no escribir documentación, pero lo hace de todos modos para futuro/as mantenedore/as.</li>
+          <li>Puede que le guste o no escribir pruebas, pero trata de aprender a hacerlo si el equipo o proyecto lo necesita.</li>
+          <li>Agradece a lo/as demás por su tiempo, esfuerzo y energía.</li>
+          <li>Puede tener fondos de escritorio coloridos.</li>
+          <li>Apoya código en producción, incluso si no lo escribieron.</li>
```

Comment:
Either

```suggestion
          <li>Apoya código en producción, incluso si no lo escribió.</li>
```

or:


```suggestion
          <li>Apoyan código en producción, incluso si no lo escribieron.</li>
```

I would be inclined towards the former since in the original version "they" seems to act as a "singular" they (genderless pronoun?).